src/components/board/board.ts

```diff
@@ -74,4 +82,17 @@ export interface Board extends Resource {
     digitalInterruptName: string,
     extra?: StructType
   ): Promise<number>;
+  /**
+   * Set power mode of the board.
+   *
+   * @param name - The name of the board.
+   * @param powerMode - The requested power mode.
+   * @param duration - The requested duration to stay in power mode.
+   */
+  setPowerMode(
+    name: string,
+    powerMode: PowerMode,
+    duration: DurationOptions,
```

Comment:
[minor] see comment related to naming

```suggestion
    duration: Duration,
```